### DISCUSSION TITLE
How does open-webui connect to ollama

### DISCUSSION THREAD

**Discussion by superChakra** [open-webui/open-webui#2953] [Q&A]
How can I access the ollama deployed model on my VM with open-webui installed?

**Comment by JuanjoJG-WATA** [comment]
Make sure Ollama is using the correct port (**11434**). If you are on Linux, use `ss-lnt` to check the open ports.

Also, go to Settings > Connections in Open-WebUI and make sure the Ollama API is activated and pointing to `http://localhost:11434`
